index.html

```diff
@@ -0,0 +1,19 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Exercise  19</title>
+        <script src="index.js">
```

Comment:
If you are referring to an external file, then the script body is ignored.
So move all the js code to the index.js file 
You should only have the following line in HTML file
`<script src="index.js"></script>`

You are almost there! Keep at it! Good job @nikhilpanchal79 💯